sloppy-deployment/test/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2.3'
```

Comment:
The sloppy-cli does support compose-format version 3 only.

Reply:
This `docker-compose.yml` is for testing only. Not to be meant as parsing config for sloppy CLI.